todo-app/src/components/FilterTodo.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+const FilterVisibility = () => {
+   const todolist = useSelector((state) => {
+     return Object.values(state.todos)
+   })
+ 
+ 
+   const handleClick =(e) => {
+      debugger
+      if(e.target.value ==='Active'){
+         return todolist.filter(todo => todo.completed === false)
+      } else if (e.target.value ==='Complete'){
+         return todolist.filter(todo=> todo.completed !== false)
```

Comment:
This can be simplified to `todolist.filter(todo=> todo.completed)` because `todo.completed`  has a value of true, and filter returns that element when it returns a truthy value